### DISCUSSION TITLE
Contributor call: 24 June 2023

### DISCUSSION THREAD

**Discussion by l3xander** [freesewing/freesewing#4260] [Support] [answered]
You are all invited to the FreeSewing Contributor Call.
Please leave a comment here or [on Discord](https://discord.freesewing.org/) to add things to the agenda.
This issue will be updated after the call with notes and to-dos.

### About
FreeSewing's contributor call is an informal hangout/chat/meeting that is open to all FreeSewing contributors and users alike.
The call is organized/hosted by @tangerineshark.
The call does not have a fixed structure or agenda items. Everyone is free to suggest agenda items, ask questions, or bring ideas.

**Details**
Location: https://discord.freesewing.org (meeting room: 🔊`voice chat`)
Date: June 24, 2023 4pm UTC, which can be found in your local time zone [here](https://savvytime.com/converter/utc-to-belgium-antwerpen-ca-los-angeles-il-chicago-ny-new-york-city-brazil-sao-paulo-united-kingdom-london-france-paris-south-africa-johannesburg-msd-india-mumbai-hkt-japan-tokyo-australia-sydney/jun-24-2023/4pm).
Or check the Events tab in Discord!
The meetings will always be at 6pm, Antwerp time.

### Agenda
...

**Comment by tangerineshark** [comment, marked as answer]
Notes

Some new folks joined - welcome Trent (ttrama on Discord, and almost certainly @/trentonTrama here, but I'll let Trent confirm that) and BestAsparagus!

Suggestions for newsletter updates

- Curvalicia/@anna-puk  shared that it could be cool to have folks share pics and/or a little bit about their sewing spaces (not forgetting the coding side here, whose workspaces were referred to as "battlestations" if memory serves 😄 )
- more pics as headers for articles might be nice for folks who skim, especially for new pattern releases
- also for people who never read the whole newsletter, a TL;DR wouldn't go amiss
- "Technique of the Week (Quarter)" would be fun - share some of the sewing expertise from Discord (this is how @tangerineshark is gonna finally share some of the amazing expertise Anthony brings to the community!)

Thrunic shared a version of the Onyx pattern - it's awesome.

Everybody wants to convince @joostdecock that the UI is pretty good and ready to share (no pressure).

@BenJamesBen vows that by next Contributor Call, he'll have constructed a test garment for Jane and Unice.

As a crew, we tinkered with the UI for pattern options (via @eriese ), so expect a PR from Enoch soon (or possibly already) that changes some of the accordion styling (e.g., adding an arrow to suggest that accordions open, getting rid of text in accordion footer but leaving a thick bottom). The consensus is pretty pleased with the result.

**Comment by joostdecock** [comment]
Closing to increase visibility of active discussions
